CHANGELOG.md

```diff
@@ -15,6 +15,13 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 6.0.0-beta.1
```

Comment:
Is it time to start the migration guide document and mention using `@rjsf/mui` instead?